sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileFormatWriter.scala

```diff
@@ -269,50 +278,57 @@ object FileFormatWriter extends Logging {
   }
 
   /**
+   * For every registered [[WriteJobStatsTracker]], call `processStats()` on it, passing it
+   * the corresponding [[WriteTaskStats]] from all executors.
+   */
+  private def processStats(
+      statsTrackers: Seq[WriteJobStatsTracker],
+      statsPerTask: Seq[Seq[WriteTaskStats]])
```

Comment:
Not sure if that's a common use case..
But if you do need to share some stats in two trackers, I can think of two solutions within the current framework:
1. in the `processStats` of the first tracker store the somewhere (e.g. catalog) and then retrieve them during `processStats` of the second tracker
2. replace the two trackers with a single, combined one (inheritance / composition)